.yamato/yamato-ci.yml

```diff
@@ -0,0 +1,67 @@
+linux:
```

Comment:
FWIW, you could parameterise these, instead of copying out a nearly-identical job definition three times.

Reply:
Now they are less identical :)